apps/fluent-tester/src/TestComponents/RadioGroupExperimental/SubtextRadioGroup.tsx

```diff
@@ -0,0 +1,21 @@
+import { RadioGroup, Radio } from '@fluentui-react-native/experimental-radio-group';
+import * as React from 'react';
+import { View } from 'react-native';
+
+export const SubtextRadioGroup: React.FunctionComponent = () => {
+  // Client's example onChange function
+  const onChange = (key: string) => {
```

Comment:
Should use React.useCallback

Reply:
Thanks for calling this out. Updated this in all of the experimental radiogroup test components.